### PR TITLE
[T3A5][W14-A1]Abel Lim

### PR DESCRIPTION
**Pull request by abellimz**

### CODE REVIEW

**Comment by abellimz** [comment]
Ready for review

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import static seedu.addressbook.common.Messages.MESSAGE_GOODBYE;
+import static seedu.addressbook.common.Messages.MESSAGE_INIT_FAILED;
+import static seedu.addressbook.common.Messages.MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE;
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+import static seedu.addressbook.common.Messages.MESSAGE_WELCOME;
+
+import java.util.List;
+
+import seedu.addressbook.commands.CommandResult;
+
```
Comment (Crphang) [inline src/seedu/addressbook/ui/Formatter.java]: You would like to add some description on what this class does.

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
+    private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
+    
+    /** Shows message(s) to the user */
```
Comment (Crphang) [inline src/seedu/addressbook/ui/Formatter.java]: Nice abstraction and use of varargs

**Comment by Crphang** [comment]
Good job in extracting the task of formatting in general

**Pull request closed without merging**